examples/neuron/inferentia.yaml

```diff
@@ -0,0 +1,55 @@
+resources:
+  cloud: aws
+  accelerators: Inferentia:6
+  disk_size: 512
+  ports: 9000
+
+envs:
+  MODEL_NAME: meta-llama/Meta-Llama-3-8B-Instruct
+  HF_TOKEN: # fill
+  NEURON_RT_VISIBLE_CORES: "0-5"
```

Comment:
If possible, we should be able to automatically set this, as we did for `CUDA_VISIBLE_DEVICE` when people specify different number of GPUs to use for a task.